lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -424,6 +426,8 @@ sub template_data_from_yaml {
                 }
                 else {
                     $testsuite_name = $spec;
+                    # Skip hidden keys
+                    next if $testsuite_name =~ /^[.]/;
```

Comment:
It might seem trivial but a duplication like this can very likely end up in inconsistencies. How about a method _skip_hidden_keys that you use in two places